src/game/rendering_graph_node.c

```diff
@@ -198,41 +194,114 @@ static const Gfx dl_silhouette_end[] = {
 struct RenderPhase {
     u8 startLayer;
     u8 endLayer;
-#ifdef OBJECTS_REJ
+// #ifdef OBJECTS_REJ
```

Comment:
a side effect of naming struct fields explicitly is that this field has to stay; opinions welcome

Reply:
Update: this is wrong and no longer needs opinions